EventKit/eventkit-cloud
# Fix feature layers

## Description of Improvement

Fixes an issue with layers key being duplicate in some configurations and missing support for picture symbols.
## How to test

Add a product which features picture symbols, also add a key to the config field of `layers: []` which would be similar to a provider that is configured to also display map previews.  The job should finish and should have pictures in the output. 

## Quality Assurance 

The following items have been updated:
 
## Screenshots



:smile:
